### DISCUSSION TITLE
My model stops updating to tensorboard after a few epochs?

### DISCUSSION THREAD

**Discussion by mm04926412** [Lightning-AI/pytorch-lightning#11320] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
This issue has appeared without warning and I'm unsure of any changes I've made that could have caused it. whenever I run a lightning model it only updates a few epochs to tensorboard before going "dark", if I stop the model and resume it then the logging continues from the new start location (step-wise) with a gap on the plot. Is there a known issue / accidental configuration option I might have applied which limits the total number of logs to tensorboard?

**Comment by rohitgr7** [comment]
I don't think there is any such limitation. Can you reproduce the issue with https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pl_examples/bug_report/bug_report_model.py?

or if you can, maybe try another logger, just to know whether this issue exists in lightning or tensorboard.
